docs/reference/hip_runtime_api/files.rst

```diff
@@ -0,0 +1,14 @@
+.. meta::
+  :description: HIP runtime files reference page.
+  :keywords: AMD, ROCm, HIP, CUDA, HIP runtime files
+
+.. _files_reference:
+
+*******************************************************************************
+Files
+*******************************************************************************
+
+The list of all files with brief descriptions and a list of all functions,
+variables, defines, enums, and typedefs with links to the files they belong to.
+
+.. include:: files_list.rst
```

Comment:
It seems like the list could be added here instead of listed in another file? 

Also, a strange thing happens when reviewing the PR readthedocs.
https://advanced-micro-devices-demo--3591.com.readthedocs.build/projects/HIP/en/3591/reference/hip_runtime_api/files.html

If you click the File List link on this page it takes you to the currently published docs (out of the PR). 
If you click on the File link in the TOC, or click the File List at the bottom under Next, it takes you to the File List in the PR.

Reply:
Fixed.